invenio_vocabularies/contrib/subjects/schema.py

```diff
@@ -23,7 +33,9 @@ class SubjectSchema(BaseVocabularySchema):
     # here.
     id = SanitizedUnicode(required=True)
     scheme = SanitizedUnicode(required=True)
-    subject = SanitizedUnicode(required=True)
+    subject = SanitizedUnicode(required=False)
```

Comment:
minor: this is a breaking change, so I would prefer if the field is still required, and we just populate with the English term (or with the default locale). This will make sure that we can smoothly transition/adapt all the UI elements and search aggregations that depend on the `subject` field.

Reply:
Oh, I forgot to change that! Our initial version of the Marc21Transformer was not aware of the subject field, but that is fixed. I'll fix that.